app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -338,6 +345,8 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
 
             settings().wasDefaultBrowserOnLastResume = settings().isDefaultBrowser()
         }
+
+        setDefaultBrowserNotificationIfNeeded()
```

Comment:
This touches IO via SharedPreferences for every resume. We may want to take some care around adding this.

cc: @mcomella

Reply:
Thanks for catching this. Can we do this on a background thread? afaict, there's nothing here that requires UI thread interaction.

Reply:
Agreed.  Thanks for catching this.  I'll do this on a background thread.